src/views/notes/Note.svelte

```diff
@@ -392,26 +392,24 @@
         </div>
         <div class="flex justify-between text-light">
           <div class="flex">
-            <div class="w-16">
-              <button class="fa fa-reply cursor-pointer" on:click|stopPropagation={startReply} />
+            <div class="w-16" on:click|stopPropagation={startReply}>
```

Comment:
What do you think about changing this element to button? It would improve accessability and semantics a little bit.
`<button class="w-16 cursor-pointer" on:click|stopPropagation={startReply}>
  <span class="fa fa-reply" aria-hidden="true" />`

https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Attributes/aria-hidden#example
https://github.com/jsx-eslint/eslint-plugin-jsx-a11y/blob/main/docs/rules/no-noninteractive-element-interactions.md

Reply:
I'll see what I can do